js_utils/axios_helpers.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+import Vue from "vue";
+import { STATUS } from "@/store/modules/flask/enums";
+
+/**
+ * Handles all HTTP GET calls from Vuex and updates system status if there was an error
+ *
+ * @param {string} whole_url - The entire URL to pass to axios.get
+ * @param {Object} action_context - The context of the Vuex action calling this function (to give this function access to the Vuex store)
+ * @return {Object} the result of the axios call
+ */
+export async function call_axios_get_from_vuex(whole_url, action_context) {
+  let result = 0;
```

Comment:
why do you think it is better to have `result = 0` instead of be `null` if there is an error?

Reply:
Currently the code in flask/actions.js had result initialized to `result = 0`

so at line 30 and 31 is as follows:- (It's not creating any issue under the circumstance when error occurs)
`  if (result.status == 200) {
    const data = result.data;`

 When we assign `result = null` then unnecessary if condition gets included as we need to validate as `if (result !== null)`, as this is a hotspot and gets invoked at every `1000` milli second, thought to retain the same performance so assigned `result = 0`. 

Please let me know is it fine to retain the `let result = 0`;

Reply:
When I handed the branch over to you, the line was just `let result;`  with no explicit assignment. Do you think there is a benefit of assigning it to `0`?

Reply:
Additionally the in playback/actions.js had result initialize to `result = 0`

`  let result = 0;
  const whole_url = `${payload.baseurl}/${payload.endpoint}`;
  result = call_axios_get_from_vuex(whole_url, this);
  if (result.status == 200) {`

Thus having `result = 0`  provides the same flow as it existed earlier without an additional check for `if (result !== null)`.